Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,47 @@
 ﻿using System;
+using System.Diagnostics;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    public static class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            string surname = "";
+            int age = 0;
+            double weight = 0;
+            double height = 0;
+
+            for (int i = 0; i <= 1; i++)
```

Comment:
A more intuitive way of writing the for loop is `for (int i = 0; i < 2; i++)`. This is currently not important, but it is good to stick with the < sign when possible unless you're doing very specific maths. The reason for this is indexing which will be discussed in later chapters. However, because you decided to use the for loop, I comment this.

Reply:
Hi, thank you for the review. Understood. I updated the code. Hope it will be okay now.